### PR TITLE
Grains n maps

### PR DESCRIPTION
**Pull request by mafrosis**
- fixed a bug in the recent refactor of grains into their own file
- merge grains from a map file with the grains defined in the profile - rather than overwriting them

### CODE REVIEW

File: saltcloud/utils/__init__.py [hunk 1]
```diff
@@ -434,7 +434,7 @@ def deploy_script(host, port=22, timeout=900, username='root',
                 if minion_grains:
                     scp_file(
                         '/tmp/grains',
-                        salt_config_to_yaml(minion_conf),
+                        salt_config_to_yaml(minion_grains),
```
Comment (s0undt3ch) [inline saltcloud/utils/__init__.py:437]: Dam! :smile:

**Comment by s0undt3ch** [comment]
Awesome @mafrosis !

**Comment by techhat** [comment]
Back in the saddle, @mafrosis! You're the man!

**Comment by mafrosis** [comment]
Thanks guys!

**Pull request merged**